themes/bootstrap5/templates/channels/channelList.phtml

```diff
@@ -23,7 +23,7 @@
   }
 ?>
 
-<?php if (empty($token)): ?>
+<?php if (empty($token) && ($displaySearchBox ?? true)): ?>
   <form action="<?=$this->url('channels-search')?>" class="channel-search form-inline">
     <?=$this->transEsc('channel_searchbox_label')?>
```

Comment:
If the search box *is* enabled, which again I'm confused on, I wonder if the label would need to be different in this context.  On the one hand, the search box does the exact same thing, so probably not.  On the other hand, would the patron be expecting the list of channels to be more record-relevant, and so the label might want to reinforce that?  IDK.

Reply:
We could add a config setting to override this, but then we're creating an extra config for a hypothetical situation. Probably best to wait until we have a use case (or, if we decide to take away the search box in tab option entirely, this becomes a non-issue).